src/styles/main.scss

```diff
@@ -1,3 +1,81 @@
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  height: 100vh;
+}
```

Comment:
you can use class to add some specific styles to body and some resets as `margin: 0` should be in reset.css